### ISSUE TITLE
Insufficient color contrast and focusable content issues with calendar

### ISSUE THREAD

**Issue by nichgray** [CDRH/lewisandclark#246]
Keeping these together so [calendar ](https://lewisandclarkjournals.unl.edu/journals/calendar)work can proceed all at once in an orderly fashion: 

- [ ] I think the color contrast issue is with the years in the calendar header
- [ ] the months container is not accessible

**Comment by erinchambers** [comment]
Nikki's spot on, the color contrast issues are with the "empty" years in the header/navigation—and also the individual dates that have no entries.
